pkg/data/usagelease.go

```diff
@@ -14,8 +14,8 @@ import (
 )
 
 const (
-	usageLeaseSkSummaryPrefix string = "Usage-Lease-Summary-"
-	usageLeaseSkDailyPrefix   string = "Usage-Lease-Daily-"
+	UsageLeaseSkSummaryPrefix string = "Usage-Lease-Summary-"
```

Comment:
Kind of bummed that this got exported but I know why you did it.  Makes sense to me.